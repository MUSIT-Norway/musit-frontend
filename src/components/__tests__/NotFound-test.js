import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';
import NotFound from '../NotFound';

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('NotFound', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<NotFound />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
