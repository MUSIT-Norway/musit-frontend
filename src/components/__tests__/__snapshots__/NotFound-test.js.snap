// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match snapshot 1`] = `
<div
  className="container"
>
  <h1>
    Not found! 404!
  </h1>
  <p>
    The route you are looking for does not exist!
  </p>
</div>
`;
